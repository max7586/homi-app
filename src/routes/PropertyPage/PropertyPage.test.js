import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import PropertyPage from './PropertyPage';
import { BrowserRouter } from 'react-router-dom'


describe('PropertyPage component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
        <BrowserRouter>
        <PropertyPage />
      </BrowserRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer.create(<BrowserRouter>
        <PropertyPage />
      </BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();  
  });


});