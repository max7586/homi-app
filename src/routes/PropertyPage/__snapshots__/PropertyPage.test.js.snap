// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropertyPage component renders the UI as expected 1`] = `
<section
  className="Section PropertyPage"
>
  <div
    className="loading"
  />
</section>
`;
